server/utils/nunjucksSetup.ts

```diff
@@ -8,6 +8,7 @@ export default function nunjucksSetup(app: express.Application, path: pathModule
       path.join(__dirname, '../../server/views'),
       'node_modules/govuk-frontend/',
       'node_modules/govuk-frontend/components/',
+      'node_modules/@ministryofjustice/frontend/',
```

Comment:
this was a bug that means the MOJ macros weren't loaded

Reply:
Can you put it in a separate commit with an explanation? It's more helpful to someone seeing this change in the future than a PR comment.

Reply:
I also kind of wonder what purpose the `/components` imports are serving in that case, if we always explicitly include `components` when we do an `import` in our templates. Probably a question for the template creators for why they did that for the `govuk` one, which I copied

Reply:
yeh i'll be honest i didn't fully work through my understanding of this code, i just wanted to get the template working.